examples/shaded-relief.js

```diff
@@ -121,7 +121,12 @@ function shade(inputs, data) {
     }
   }
 
-  return {data: shadeData, width: width, height: height};
```

Comment:
`ImageData` has required property `colorSpace`, defaults to `'srgb'`.
https://developer.mozilla.org/en-US/docs/Web/API/ImageData/colorSpace